source/__tests__/main.index.test.js

```diff
@@ -0,0 +1,106 @@
+//main.index.test.js
+
+import { getCurrentDate, formatDateToMonthDayYear, getMonthYear, formatDateToYYYYMMDD, isCurrentDate, isCurrentMonth, isInFuture } from "./date.util.js";
+import "./main.index.js";
```

Comment:
Not too sure how the import is being used, but the intention was that the functions in the file would be tested here (createRow, createDay, changeMonth, renderCalendar etc.)